src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public class FigureSupplier {
+    public Figure getRandomFigure(){

```

Comment:
The method `getRandomFigure` is missing a return type in its signature. It should return a `Figure` object.